data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	Tavish sighs, "Splendid work, though I do think we should run now."`
+			`	He turns to you. "We'll meet you at your ship."`
+			`	The two of them then sprint off before the authorities have time to respond.`
+			choice
+				`	(Head back to your ship.)`
+			`	Tavish and Kormak are waiting for you near your ship. They only look slightly disheveled in their overdressed suits.`
+			`	"Excellent," Tavish says as he claps his hands together. "Deadman's Cove awaits us. Let's waste no time, Captain <last>."`
+			choice
+				`	"I never agreed to take you to Deadman's Cove!"`
+				`	"You're not the one giving orders here, and Deadman's Cove was not part of our agreement."`
+				`	"Fine, but first you need to answer some questions."`
+					goto questions
+				`	"You just assaulted an official! I don't know who you are, but I want none of this. I am out."`
+					goto quit
+
+			`	Tavish holds out his hands, "True! Most true! And we categorically thank you for your bottomless hospitality so far."`
+			`	"Bottomless." Kormak affirms.`
+			`	"But our grand adventure has just begun and surely you don't want to leave now before you've seen how it all unfolds." Tavish gives you a winning smile.`
+			`	"What's a few extra jumps compared to learning the most profound secrets of the universe? Come now, old chap, let's not allow some minor discomfort deter you from this adventure!"`
+			choice
+				`	"Fine, but first you need to answer some question."`
+					goto questions
+				`	"Nope. I don't know what this is, but I am out."`
+
+			label quit
+			`	Tavish frowns and sighs, but says nothing. He looks at Kormak and then begins to wander away.`
+			`	Kormak glares at you and then says, "Could have had fun."`
+			`	The large man turns and follows Tavish back into Clark's wild spaceport.`
+				decline
+			
+			label questions
+			`	"Of course." Tavish nods emphatically, "We assuredly are an open book."`
+			`	Kormak delivers a dissenting grunt.`
+			`	"Uh, mostly open." Tavish quickly amends.`
+
+			label questions2
+			choice
+				`	"Just who exactly are you?"`
+					goto who1
+					to display
+						not "who1"
+				`	"What sort of science are you doing on New India?"`
+					goto india1
+					to display
+						not "india1"
+				`	"What is this equipment you need so badly?"`
+					goto equipment1
+					to display
+						not "equipment1"
+				`	"I have no more questions. But from now on I'm in charge. Now, let's go to Deadman's Cove."`
+					goto end
+
+			label who1
+			action
+				set "who1"
+			`	"Just as I said earlier, we're Tavish and Kormak." Tavish starts.`
+			`	"Kormak and Tavish." Kormak corrects him.`
+			`	"And," Tavish says ignoring Kormak, "we are doctors, researchers, and most importantly, explorers!"`
+			`	Tavish pauses and levels a look at Kormak, "At least, I think we're both doctors."`
+			`	"Very small school in Deep. Likely not heard of." Kormak says nonchalantly.`
```

Comment:
He's from the Deep. Not a planet called Deep.

Reply:
After reading more and seeing Kormak's speech patterns, "in Deep" works (although because there is a planet called Deep, it's a little confusing). I'd still use the comma, though:
```suggestion
			`	"Very small school in Deep. Likely not heard of," Kormak says nonchalantly.`
```